src/history/hs_rec.c

```diff
@@ -219,8 +219,8 @@ __hs_insert_record(WT_SESSION_IMPL *session, WT_CURSOR *cursor, WT_BTREE *btree,
     }
 
     if (ret == 0)
-        WT_ERR(__hs_delete_reinsert_from_pos(
-          session, cursor, btree->id, key, tw->start_ts + 1, true, error_on_ooo_ts, &counter));
+        WT_ERR(__hs_delete_reinsert_from_pos(session, cursor, btree->id, key, tw->start_ts + 1,
+          true, false, error_on_ooo_ts, &counter));
```

Comment:
I think false and true are interchanged. Previous logic error_on_ooo_ts is being passed as true, now it became false.

Reply:
No. In the PR, an extra variable is passed in between the two to represent whether it is an out of order tombstone correction or not? The original value for err_on_ooo_ts is passed the same.